.github/workflows/ci_cron_weekly.yml

```diff
@@ -60,6 +60,11 @@ jobs:
             python: '3.12'
             toxenv: py312-test-devinfra
 
+          - name: Python 3.12 on macOS (x86_64) with all optional dependencies
+            os: macos-13
```

Comment:
Typo?
```suggestion
            os: macos-12
```

Reply:
actually not a typo: macos-13 is the latest image available that uses the `x86_64` arch, so I figure it'll probably be around for a bit longer, it just never got the spotlight and was never aliased to `macos-latest`.